bve/src/parse/mesh/instructions/execution.rs

```diff
@@ -72,6 +73,7 @@ impl Instruction {
     }
 }
 
+// Weird that this function has an output argument while `flat_shading` returns its output. Should be consistent
```

Comment:
The idea was to.not have to create an intermediate vector but I should probably just use SmallVec and call it a day

Reply:
I don't think it'd be a problem. This is just loading code

Reply:
Yeah you right

Reply:
Solved in 2c6c0d7625440ce3dc0c9da4a1f76c5b83821b6a